### PR TITLE
Remove need for DEBUG var to avoid developers debugging blindly

### PR DESCRIPTION
**Pull request by sedubois**

### CODE REVIEW

File: server/middleware/error_handler.js [hunk 1]
```diff
@@ -53,16 +53,15 @@ module.exports = (err, req, res, next) => {
...
-    if(req.app.set('env') === 'production') {
+  switch (process.env.NODE_ENV) {
+    case 'production':
       sendErrorByEmail(req, err);
-    }
-
-    console.error('Error Express : ', err);
-    if (err.stack) console.log(err.stack);
+      break;
+    case 'development':
```
Comment (asood123) [inline server/middleware/error_handler.js]: What happens in Staging and circleci?

Comment (sedubois) [inline server/middleware/error_handler.js]: The stack then isn't printed, as was already the case whenever `DEBUG` wasn't defined.

Comment (sedubois) [inline server/middleware/error_handler.js]: Actually it did print it for nightwatch tests.

Comment (asood123) [inline server/middleware/error_handler.js]: would be nice to have for circleci - those things are hard to debug as it is.

**Pull request merged**